core/assets/game.play

```diff
@@ -1,25 +1,46 @@
 # Hamorigami Gameplay Setup
 
-# DAY 1 - nothing much is happening here
-#
-# this is a comment
-day 1
+# day 0 - story set up
+# evening only, clear skies
 
```

Comment:
unfortunately, we do not support "half days" right now (as stupid as it sounds)

Let me add this as a feature. It is actually a good idea.